aws/resource_aws_emr_instance_group_test.go

```diff
@@ -129,46 +188,23 @@ func testAccCheckAWSEmrInstanceGroupExists(n string, v *emr.InstanceGroup) resou
 	}
 }
 
+func testAccAWSEMRInstanceGroupResourceImportStateIdFunc(resourceName string) resource.ImportStateIdFunc {
+	return func(s *terraform.State) (string, error) {
+		rs, ok := s.RootModule().Resources[resourceName]
+		if !ok {
+			return "", fmt.Errorf("Not found: %s", resourceName)
+		}
+
+		log.Println(fmt.Sprintf("%s/%s", rs.Primary.Attributes["cluster_id"], rs.Primary.ID))
+		return fmt.Sprintf("%s/%s", rs.Primary.Attributes["cluster_id"], rs.Primary.ID), nil
+	}
+}
+
 const testAccAWSEmrInstanceGroupBase = `
 provider "aws" {
```

Comment:
Not caused by this pull request, but most provider acceptance test configurations should omit `provider` configurations (unless explicit multi-region testing)